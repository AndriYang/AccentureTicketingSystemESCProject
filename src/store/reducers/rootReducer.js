import authReducer from './authReducer'
import projectReducer from './projectReducer'
import { combineReducers } from 'redux'
import { firestoreReducer } from 'redux-firestore'
import { firebaseReducer } from 'react-redux-firebase'
import dataReducer from './dataReducer'
import formReducer from './formReducer'
import replyReducer from './replyReducer'
import ticketReducer from './ticketReducer'

const rootReducer = combineReducers({
  auth: authReducer,
  project: projectReducer,
  firestore: firestoreReducer,
  firebase: firebaseReducer,
  data: dataReducer,
  form: formReducer,
  reply: replyReducer,
  ticket: ticketReducer
});

export default rootReducer
