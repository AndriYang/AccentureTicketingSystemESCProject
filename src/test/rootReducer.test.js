import rootReducer from '../store/reducers/rootReducer'

describe('With snapshots', () => {
  it('should return initial state', () =>{
    expect(rootReducer(undefined, {})).toMatchSnapshot();
  })

  
})
