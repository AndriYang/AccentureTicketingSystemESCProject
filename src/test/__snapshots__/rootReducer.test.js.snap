// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With snapshots should return initial state 1`] = `
Object {
  "auth": Object {
    "authError": null,
  },
  "data": Object {},
  "firebase": Object {
    "auth": Object {
      "isEmpty": true,
      "isLoaded": false,
    },
    "authError": null,
    "data": Object {},
    "errors": Array [],
    "isInitializing": false,
    "listeners": Object {
      "allIds": Array [],
      "byId": Object {},
    },
    "ordered": Object {},
    "profile": Object {
      "isEmpty": true,
      "isLoaded": false,
    },
    "requested": Object {},
    "requesting": Object {},
    "timestamps": Object {},
  },
  "firestore": Object {
    "composite": undefined,
    "data": Object {},
    "errors": Object {
      "allIds": Array [],
      "byQuery": Object {},
    },
    "listeners": Object {
      "allIds": Array [],
      "byId": Object {},
    },
    "ordered": Object {},
    "queries": Object {},
    "status": Object {
      "requested": Object {},
      "requesting": Object {},
      "timestamps": Object {},
    },
  },
  "form": Object {},
  "reply": Object {},
  "ticket": Object {
    "projects": Array [
      Object {
        "content": "blah blah blah",
        "id": "1",
        "title": "help me find peach",
      },
      Object {
        "content": "blah blah blah",
        "id": "2",
        "title": "collect all the start",
      },
      Object {
        "content": "blah blah blah",
        "id": "3",
        "title": "egg hunt with me",
      },
    ],
  },
}
`;
